### PR TITLE
feat: release 3.3.0

### PR DESCRIPTION
**Pull request by leslie-tsang**
### Description

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/CHANGELOG.md [hunk 1]
```diff
@@ -68,6 +69,38 @@ title: CHANGELOG
...
+
+- :sunrise: fault-injection 插件支持请求头注入: [#9039](https://github.com/apache/apisix/pull/9039)
+- :sunrise: 提供在其他插件中引用 proxy-rewrite 插件中路由改写捕捉到的变量支持: [#9112](https://github.com/apache/apisix/pull/9112)
+- :sunrise: limit-count 插件提供 `username` 与 `ssl` redis 认证方式: [#9185](https://github.com/apache/apisix/pull/9185)
+
+### Bugfixes
+
+- 修复 etcd 数据同步异常: [#8493](https://github.com/apache/apisix/pull/8493)
+- 修复在 `core.request.add_header` 中的无效缓存: [#8824](https://github.com/apache/apisix/pull/8824)
+- 修复由健康检查实现引起的高 CPU 和内存占用: [#9015](https://github.com/apache/apisix/pull/9015)
```
Comment (soulbird) [inline docs/zh/latest/CHANGELOG.md:94]: 修复由健康检查引起的高 CPU 和内存占用

File: docs/zh/latest/CHANGELOG.md [hunk 2]
```diff
@@ -68,6 +69,38 @@ title: CHANGELOG
...
+## 3.3.0
+
+### Change
+
+- 默认路由从 `radixtree_uri` 修改为 `radixtree_host_uri`: [#9047](https://github.com/apache/apisix/pull/9047)
+- CORS 插件将会在 `allow_origin` 不为 `*` 时默认添加 `Vary: Origin` 响应头: [#9010](https://github.com/apache/apisix/pull/9010)
+
+### Core
+
+- :sunrise: 支持在 secrets manager 中存储路由证书: [#9247](https://github.com/apache/apisix/pull/9247)
```
Comment (soulbird) [inline docs/zh/latest/CHANGELOG.md:81]: 支持将路由证书存储在 secrets manager 中

Comment (leslie-tsang) [reply docs/zh/latest/CHANGELOG.md]: Done

File: docs/zh/latest/CHANGELOG.md (hunk 1 above)
Comment (leslie-tsang) [reply docs/zh/latest/CHANGELOG.md]: Done

**Review by soulbird — DISMISSED** [review]

**Review by monkeyDluffy6017 — DISMISSED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -68,6 +69,38 @@ title: Changelog
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 3.3.0
+
+### Change
+
+- Change the default router from `radixtree_uri` to `radixtree_host_uri`: [#9047](https://github.com/apache/apisix/pull/9047)
```
Comment (moonming) [inline CHANGELOG.md:76]: this is a break change

**Review by moonming — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (leslie-tsang) [reply CHANGELOG.md:76]: Would it be better to add an underline or note ?

**Review by AlinsRan — DISMISSED** [review]

**Review by soulbird — APPROVED** [review]

**Review by monkeyDluffy6017 — APPROVED** [review]

**Review by moonming — APPROVED** [review]

**Pull request merged**